vector/src/main/res/values/strings.xml

```diff
@@ -3014,7 +3022,11 @@
     <string name="location_timeline_failed_to_load_map">Failed to load map</string>
     <string name="location_share_live_enabled">Live location enabled</string>
     <string name="location_share_live_started">Loading live location…</string>
+    <string name="location_share_live_ended">Live location ended</string>
+    <string name="location_share_live_view">View live location</string>
+    <string name="location_share_live_until">Live until %1$s</string>
```

Comment:
Maybe add a comment to explain what will replace the template, with an example

Reply:
Done in [e9d9319](https://github.com/vector-im/element-android/pull/5989/commits/e9d93194f15f58c3ce10e132b6fc244de4c8f5bb).